include/SearchForm/SearchForm2.php

```diff
@@ -1196,7 +1196,13 @@ public function generateSearchWhere($add_custom_fields = false, $module = '')
                                     if (!empty($parms['subquery'])) {
                                         $selectCol = $this->getSelectCol($parms['subquery']);
                                     }
-                                    $where .= "{$db_field} $in (select $selectCol from ({$parms['subquery']} " . $this->seed->db->quoted($field_value . '%') . ") {$field}_derived)";
+                                    if ($operator == 'subquery' && $parms['type'] !== 'enum') {
```

Comment:
Please look over queries taking into consideration SQL Injection - using the quoted() function where appropriate.